### DISCUSSION TITLE
Is there a way to store Resharper specific colors into a theme?

### DISCUSSION THREAD

**Discussion by PiotrSobiegraj** [definedrisk/NightOwl-VS-Theme#2] [Q&A] [answered]
Hi,
as in subject. By default "Resharper Inlay Hint" and "Resharper Debugger DataTip" display items are completely unreadable with your theme. I managed to make them readable via Tools->Options->Environment->Fonts and Colors as in picture. I'm wondering is there a way to persist them inside a theme?
![Screenshot 2022-02-22 024137](https://user-images.githubusercontent.com/1130719/155047861-8ae2ecef-f7eb-4120-a7b6-85f43279e803.png)

**Comment by definedrisk** [comment, marked as answer]
Hello, I would like to try... I do not currently have the *ReSharper* extension installed. Instead, if you could please do `Tools->Import and Export Settings...` and then export and send me just the *Fonts and Colors* settings as a `.vssettings` file I will hopefully be able to add to the theme. Before you do this, please select your choice of colors for all of the ReSharper display items that you think are best (when using this theme). Hopefully you can test for me and then I will release a new issue and reference you as providing the help.

**Reply by PiotrSobiegraj** [reply]
Hi, sorry for delay. Here you have these setting. With extra extension as vssettings is  not supported. I ting that this color set is fine.
[Exported-2022-03-08.vssettings.txt](https://github.com/definedrisk/NightOwl-VS-Theme/files/8209119/Exported-2022-03-08.vssettings.txt)
![Screenshot 2022-03-08 204241](https://user-images.githubusercontent.com/1130719/157313016-51c8dc54-3ad2-4ae7-a021-635ed162024b.png)

**Reply by definedrisk** [reply]
I'm committing the revised code. Please can you see if it works for you? Change your previously two adjusted *"Display item"* *foreground* and *background* colour to `default`. I think from your screenshot this was "ReSharper Inlay Hint" - however the name used in the xml theme file seems to be "ReSharper Debugger DataTip" and "ReSharper Parameter Name Hint".

I also noted that the exported files (as you show above) list colors as BGR whereas the XML theme uses RGB.
